openreview/venue/webfield/areachairsWebfield.js

```diff
@@ -0,0 +1,67 @@
+// Webfield component
+const VENUE_ID = ''
+const SHORT_PHRASE = ''
+const SUBMISSION_ID = ''
+const REVIEWERS_NAME = ''
+const AREA_CHAIRS_NAME = ''
+const OFFICIAL_REVIEW_NAME = ''
+const META_REVIEW_NAME = ''
+const SUBMISSION_NAME = ''
+const REVIEW_RATING_NAME = 'rating'
+const REVIEW_CONFIDENCE_NAME = 'confidence'
+const META_REVIEW_CONFIDENCE_NAME = 'recommendation'
+const reviewerAssignmentTitle=null
+
+const areaChairsId=`${VENUE_ID}/${AREA_CHAIRS_NAME}`
+const reviewerGroup=`${VENUE_ID}/${REVIEWERS_NAME}`
+
+// query search is disabled if filterOperators or propertiesAllowed is missing
+const filterOperators = ['!=','>=','<=','>','<','=']; // sequence matters
+const propertiesAllowed ={
+    number:['note.number'],
+    id:['note.id'],
+    title:['note.content.title'],
+    author:['note.content.authors','note.content.authorids'], // multi props
+    keywords:['note.content.keywords'],
+    reviewer:['reviewProgressData.reviewers'],
+    numReviewersAssigned:['reviewProgressData.numReviewers'],
+    numReviewsDone:['reviewProgressData.numSubmittedReviews'],
+    ratingAvg:['reviewProgressData.averageRating'],
+    ratingMax:['reviewProgressData.maxRating'],
+    ratingMin:['reviewProgressData.minRating'],
+    confidenceAvg:['reviewProgressData.averageConfidence'],
+    confidenceMax:['reviewProgressData.maxConfidence'],
+    confidenceMin:['reviewProgressData.minConfidence'],
+    replyCount:['reviewProgressData.forumReplyCount'],
+    recommendation:['metaReviewData.recommendation'],
+    ranking:['metaReviewData.ranking']
+}
+
+return {
+  component: 'AreaChairConsole',
+  version: 2,
```

Comment:
i think version here should be 1
it's to differentiate the component like AreaChairConsoleVersion2 from AreaChairConsole

the property for api version is apiVersion

Reply:
fixed